website/blog/2023-01-12-version-071.md

```diff
@@ -0,0 +1,246 @@
+---
+title: 'React Native 0.71: TypeScript by Default, Flexbox Gap, and more...'
+authors: [lunaleaps, mattcarrollcode, cortinico, kelset]
+tags: [announcement, release]
+date: 2023-01-12
+---
+
+Today we’re releasing React Native version 0.71. This is a feature-packed release which includes TypeScript by default, Flexbox gap support, new web-inspired props, developer experience improvements, updates to the New React Native Architecture, and more!
+
```

Comment:
It probably makes sense to truncate here and link to the different titles
```suggestion

<!--truncate-->

```

Reply:
here's a reference of how it should be https://github.com/facebook/react-native-website/blob/main/website/blog/2023-01-03-typescript-first.md?plain=1#L16

Reply:
titles that should be listed pre-truncate:
* TypeScript by default
* Simplifying layouts with Flexbox gap
* Web-inspired props for accessibility, styles, and events
* Restoring PropTypes
* Developer Experience Improvements
* Breaking changes